nevergrad/functions/photonics/photonics.py

```diff
@@ -27,7 +27,7 @@ def bragg(X: np.ndarray) -> float:
     n = np.concatenate(([1], np.sqrt(X[0:bar]), [1.7320508075688772]))
     Type = np.arange(0, bar + 2)
```

Comment:
to help have uniform and readable code, in the future I would suggest you avoid using capitalized names for instances like `Type` (cf [pep8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)). In this case naming it `type` would actually be worth though, since this is a reserved name :D